spec/acceptance/message_actions_spec.rb

```diff
@@ -193,7 +211,7 @@
     )
     expect(nothing.messages.size).to eq 0
 
-    sleep(5)
+    sleep(7)
```

Comment:
I've added a commit in #32 with the comment in the spec. 

I made the commit there, since that's where I detailed/explained the original change in the PR.